IPython/core/tests/test_formatters.py

```diff
@@ -103,7 +103,7 @@ def set_fp(p):
 
 def test_for_type():
     f = PlainTextFormatter()
-    
```

Comment:
I think these got automatically added by the pre-commit.

Reply:
no worries.